backends/p4tools/cmake/common.cmake

```diff
@@ -18,7 +18,7 @@ macro(p4tools_obtain_z3)
 
   if(TOOLS_USE_PREINSTALLED_Z3)
     # We need a fairly recent version of Z3.
-    set(Z3_MIN_VERSION "4.8.14")
+    set(Z3_MIN_VERSION "4.8.12")
```

Comment:
How is this connected to namespaces?

Reply:
Thanks for catching this! It's a mistake.

Reply:
Fixed.